### ISSUE TITLE
DOCS: Optimistic/outdated that `make test` "should take less than a minute"?

### ISSUE THREAD

**Issue by HenrikBengtsson** [etal/cnvkit#13]
The `README.rst` says:

> If this pipeline completes successfully (it should take less than a minute), you've installed CNVkit correctly.

Is this an outdated benchmark, because on a decent Linux machine, this took ~10 minutes for me.

**Comment by HenrikBengtsson** [comment]
Actually, could it be because `make` (now?) runs `make all`?

**Comment by etal** [comment]
It's definitely an outdated benchmark -- I've added a few more samples to process since then and also generate "diagram" and "scatter" plots for every sample now. More importantly, I usually run `make -j` to parallelize it, and on an 8-core Linux workstation that does finish in about a minute. I just timed it on my laptop: 6 minutes, 3 seconds.

I'll update the sentence to "a few minutes" and mention the `-j` flag.
